include/exadg/compressible_navier_stokes/spatial_discretization/kernels_and_operators.h

```diff
@@ -981,7 +981,8 @@ class ViscousOperator
 
     IP::calculate_penalty_parameter<dim, Number>(array_penalty_parameter,
                                                  *matrix_free,
-                                                 data.dof_index);
+                                                 data.dof_index,
+                                                 data.quad_index);
```

Comment:
Add parameter `ViscousOperatorData::quad_index_standard` (initialized in `CompNS::Operator` with `quad_index_standard`) and use it here. The reason is that the present PR should not change numerics in my opinion. We could redefine this in a follow-up PR.

Note that nothing has to be done for `CompNS::CombinedOperatorData`! The reason is that the `CompNS` module does not use the kernel-design (that is used for the `ConvDiff` and `IncNS` modules). 

Introducing the kernel design for `CompNS` for consistency with other ExaDG modules would have to be done in a follow-up PR.